experimental/segmenter/src/rule_segmenter.rs

```diff
@@ -216,6 +238,39 @@ impl<'l, 's, Y: RuleBreakType<'l, 's>> RuleBreakIterator<'l, 's, Y> {
         }
         true
     }
+
+    /// Return the status value of break boundary.
+    /// If segmenter isn't word, always return RuleStatusType::None
+    pub fn rule_status(&self) -> RuleStatusType {
+        if self.result_cache.first().is_some() {
+            // Dictionary type (CJ and East Asian) is letter.
+            return RuleStatusType::Letter;
+        }
+        if self.boundary_property == 0 {
+            // break position is SOT / Any
+            return RuleStatusType::None;
+        }
+        if let Some(value) = self
+            .data
+            .rule_status_table
+            .0
+            .get((self.boundary_property - 1) as usize)
+        {
+            match value {
+                1 => RuleStatusType::Number,
+                2 => RuleStatusType::Letter,
+                _ => RuleStatusType::None,
+            }
+        } else {
+            RuleStatusType::None
+        }
+    }
```

Comment:
nit: could be simplified

```suggestion
        match self
            .data
            .rule_status_table
            .0
            .get((self.boundary_property - 1) as usize)
        {
            Some(1) => RuleStatusType::Number,
            Some(2) => RuleStatusType::Letter,
            _ => RuleStatusType::None,
        }
    }
```